lib/blocks.js

```diff
@@ -860,6 +867,23 @@ var blockStarts = [
     }
 ];
 
+const reValidTableDelimiter = /^:?-+:?$/;
```

Comment:
For the delimiter row to be valid, all of the cells in it must be made up of a string of dashes with a colon before or after to change the alignment of that row. They can start and end with whitespace, but that's removed before checking with the regex

Reply:
All the other regexes are between line 14095-14137. Should we put this one there too?

Reply:
Good call. I started defining the regexes with their corresponding functions in `lib/inlines.js`, but I guess I never did that for `lib/blocks.js`